src/page/LoginPage/hooks/usePassword.tsx

```diff
@@ -0,0 +1,28 @@
+import { useMemo, useState } from 'react';
+
+const usePassword = (password: string) => {
+  const [value, setValue] = useState<string>(password);
+  const [error, setError] = useState<string>('');
+  const NUM = 8;
+
+  const isValid = useMemo(() => {
+    return value.trim() !== '' && value.length >= NUM;
+  }, [value]);
+
+  const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(e.target.value);
+  };
+
+  const onBlur = () => {
+    if (value.trim() === '') {
+      setError('PW를 입력해주세요.');
+    } else if (value.length < NUM) {
+      setError('PW는 최소 8글자 이상이어야 합니다.');
+    } else {
+      setError('');
+    }
+  };
```

Comment:
유효성 검증 함수는 유틸성 함수로 볼 수 있어보여요.
외부에 에러 메시지를 반환하는 유효성 검증 함수를 바깥에 선언해 사용해보는건 어떻게 생각하시나요?

Reply:
그리고 에러 메시지도 상수로 관리하면 유지보수에 더 좋아보여요 😊

Reply:
여기서 선언한 상수값을 사용할 수 있어보이네요 😊

```suggestion
  const onBlur = () => {
    if (value.trim() === '') {
      setError('PW를 입력해주세요.');
    } else if (value.length < NUM) {
      setError(`PW는 최소 ${NUM}글자 이상이어야 합니다.`);
    } else {
      setError('');
    }
  };
```

Reply:
'외부에 에러 메시지를 반환하는 유효성 검증 함수를 바깥에 선언해 사용해보는건 어떻게 생각하시나요?' 라고 질문 주셨는데 
훅은 상태를 관리하는 역할에 충실하게 하고 유효성 검증은 상태관리에 필요한 부가적인 함수이기 때문에 분리하는 방향으로 진행하는게 괜찮은 방향이라는 의미인가요?

Reply:
훅과 일반 함수의 차이점을 함께 고민해보시면 좋아보여요
어떤걸 훅으로 작성하고 어떤걸 일반 함수로 작성할 수 있을까요?